packages/api/cms-api/src/user-permissions/decorators/content-scopes.decorator.ts

```diff
@@ -0,0 +1,9 @@
+import { createParamDecorator, ExecutionContext } from "@nestjs/common";
+
+import { getRequestFromExecutionContext } from "../../common/decorators/utils";
+import { ContentScope } from "../interfaces/content-scope.interface";
+
+export const ContentScopes = createParamDecorator((data: unknown, context: ExecutionContext): ContentScope[] | undefined => {
```

Comment:
Do we even need this? Is it ever necessary to access the content scopes in a GraphQL resolver when it's not the scope-arg?

Reply:
probably we don't need it. better leave it out for now?

Reply:
Removed: f1bf0ae868b267b8f29c4eefc5e65062d6775b24